src/HealthChecks.AzureStorage/AzureBlobStorageHealthCheck.cs

```diff
@@ -1,85 +1,46 @@
-using System.Collections.Concurrent;
 using Azure.Core;
 using Azure.Storage.Blobs;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Options;
 
 namespace HealthChecks.AzureStorage
 {
     public class AzureBlobStorageHealthCheck : IHealthCheck

```

Comment:
If I could propose a breaking change for the eventual 7.0.0 version, it would be making these health check classes internal, just so we don't have to expose their classes and ctors

Reply:
This contradicts the general HC design from this repo.

Reply:
Then certainly we can leave it `public`